src/core/hle/service/sockets/sfdnsres.h

```diff
@@ -19,6 +20,11 @@ class SFDNSRES final : public ServiceFramework<SFDNSRES> {
 
 private:
     void GetAddrInfoRequest(Kernel::HLERequestContext& ctx);
+    void GetAddrInfoRequestWithOptions(Kernel::HLERequestContext& ctx);
+
+    std::pair<u32, s32> GetAddrInfoRequestImpl(Kernel::HLERequestContext& ctx);
+    const std::vector<u8> SerializeAddrInfo(const addrinfo* addrinfo, s32 result_code,
```

Comment:
```suggestion
    std::vector<u8> SerializeAddrInfo(const addrinfo* addrinfo, s32 result_code,
```

Also `<string_view>`, `<utility>` and `<vector>` need to be included